lib/source-map-resolve-node.js

```diff
@@ -71,8 +71,45 @@ function resolveSourceMapSync(code, codeUrl, read) {
 }
 
 var dataUriRegex = /^data:([^,;]*)(;[^,;]*)*(?:,(.*))?$/
+
+/**
+ * The media type for JSON text is application/json.
+ * 
+ * {@link https://tools.ietf.org/html/rfc8259#section-11 | IANA Considerations }
+ * 
+ * `text/json` is non-standard media type
+ */
 var jsonMimeTypeRegex = /^(?:application|text)\/json$/
 
+/**
+ * JSON text exchanged between systems that are not part of a closed ecosystem
+ * MUST be encoded using UTF-8.
+ * 
+ * {@link https://tools.ietf.org/html/rfc8259#section-8.1 | Character Encoding}
+ */
+var jsonCharacterEncoding = "utf-8"
+
+function base64ToBuf(b64) {
+  var binStr = atob(b64)
+  var len = binStr.length
+  var arr = new Uint8Array(len)
+  for (var i = 0; i < len; i++) {
+    arr[i] = binStr.charCodeAt(i)
+  }
+  return arr
+}
+
+function decodeBase64String(b64) {
+  if (typeof TextDecoder === "undefined" || typeof Uint8Array === "undefined") {
+    return atob(b64)
```

Comment:
I removed the warning because I’m worried it would spam the console. I don’t think many people will end up here though.